website/docs/docs/build/incremental-microbatch.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "About microbatch incremental models"
+description: "Learn about the 'microbatch' strategy for incremental models."
+id: "incremental-microbatch"
+---
+
+# About microbatch incremental models <Lifecycle status="beta" />
+
+:::info Microbatch
+
+The `microbatch` strategy is available in [dbt Cloud Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) and dbt Core v1.9.
+
+Read and participate in the discussion: [dbt-core#10672](https://github.com/dbt-labs/dbt-core/discussions/10672)
+
+:::
+
+## What is microbatch?
+
+Incremental models in dbt are a [materialization](/docs/build/materializations) designed to efficiently update your data warehouse tables by only transforming and loading _new or changed data_ since the last run. Instead of reprocessing an entire dataset every time, incremental models append, update, or replace rows in the existing table with the new data just processed. This can significantly reduce the time and resources required for your data transformations.
+
+Microbatch incremental models make it possible to process transformations on large time-series datasets with efficiency and resiliency. When dbt runs a microbatch model — whether for the first time, during incremental runs, or during manual backfills — it will split the processing into multiple queries (or "batches"), based on the `event_time` column you configure.
+
+Where other incremental strategies operate only on "old" and "new" data, microbatch models treat each "batch" of data as a unit that can be built or replaced on its own. Each batch is independent and <Term id="idempotent" />. This is a powerful abstraction that makes it possible for dbt to run batches separately — in the future, concurrently — and to retry them independently.
+
+### Available configs
+
+- `event_time` - The column indicating "at what time did the row occur" (for both your microbatch model and its direct parents)
```

Comment:
maybe a callout that `event_time` and `begin` need to be in UTC? (also true of the CLI args `--event-time-start/end`

Reply:
or maybe having the "timezone"s section is better!